oarepo_requests/ui/theme/assets/semantic-ui/js/oarepo_requests_ui/request-detail/components/ActionButtons.jsx

```diff
@@ -0,0 +1,68 @@
+import React from "react";
+
+import { i18next } from "@translations/oarepo_requests_ui/i18next";
+import { Button } from "semantic-ui-react";
+import _isEmpty from "lodash/isEmpty";
+import axios from "axios";
+
+import { ConfirmModal } from ".";
+
+const callApi = async (url, method = "GET", data = null) => {
+  if (_isEmpty(url)) {
+    console.log("URL parameter is missing or invalid.");
+  }
+  data = { data: data };
+  if (_isEmpty(data.data?.payload?.content)) {
+    data = null;
+  }
+  return axios({
+    url: url,
+    method: method,
+    headers: {
+      'Content-Type': 'application/json',
+      'Accept': 'application/vnd.inveniordm.v1+json'
+    },
+    ...data
+})};
+
+export const ActionButtons = ({ request }) => {
```

Comment:
Could be refactored to use same action components as request modal

Reply:
It could, but the functionality is different. Much less simpler, since this is just potentially four action buttons rendered per one request and the rendered modal doesn't have any customSubmitHandler like RequestModal. #34 fixes issues with RequestModal.